target/linux/lantiq/xrx200/config-5.4

```diff
@@ -25,7 +26,8 @@ CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_POLLDEV=y
 CONFIG_INTEL_XWAY_PHY=y
 # CONFIG_ISDN is not set
-CONFIG_LANTIQ_XRX200_LEGACY=y
+CONFIG_LANTIQ_XRX200=y
+CONFIG_LEDS_TRIGGER_HEARTBEAT=y
```

Comment:
https://github.com/openwrt/openwrt/commit/22061b28802a1aefa0cc4e1a1cdea69b4e432e5d

Reply:
some make kernel_menu(or old)config CONFIG_TARGET=subtarget also added to this file:
CONFIG_CRYPTO_LIB_POLY1305_RSIZE=2
CONFIG_MIPS_CBPF_JIT=y

Reply:
you are right - I'll need to remove `CONFIG_LEDS_TRIGGER_HEARTBEAT=y` from here, thanks for this hint!

the other two seem unrelated to my pull request
can you please take care of them by sending a patch/pull request?